src/styles/main.scss

```diff
@@ -0,0 +1,76 @@
+@import "./mixins";
+@import "./variables";
+@import "./keyframes";
+@import "./reset.scss";
+
+.content {
+  @include flexCenter;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
```

Comment:
8. [BEM & STYLES] - Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.